framework/src/outputs/Output.C

```diff
@@ -82,12 +79,15 @@ validParams<Output>()
 MultiMooseEnum
 Output::getExecuteOptions(std::string default_type)
 {
-  // Build the string of options
-  std::string options = "none=0x00 initial=0x01 linear=0x02 nonlinear=0x04 timestep_end=0x08 "
-                        "timestep_begin=0x10 final=0x20 failed=0x80";
-
-  // The numbers associated must be in sync with the ExecFlagType in Moose.h
-  return MultiMooseEnum(options, default_type);
+  /*
+  ::mooseDeprecated("The 'getExecuteOptions' was replaced by the ExecFlagEnum class because MOOSE "
+                    "was updated to use this for the execute flags and the new function provides "
+                    "additional arguments for modification of the enum.");
+  */
```

Comment:
Remove?

Reply:
These will be enabled after the apps have been updated, so non tested apps will get a useful message. I can pull the comments out and add them back in a non-comments when the update is finished, it was just easier to put them in while I was working on this and then comment them out to get everything working.